plugins/entity-feedback-common/src/index.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Common functionalities for the entity-feedback plugin.
+ *
+ * @packageDocumentation
+ */
+
+/**
+ * @public
+ */
+export interface Rating {
+  entityRef: string;
+  rating: string;
+  userRef: string;
+}
+
+/**
+ * @public
+ */
+export interface Response {
```

Comment:
Wonder if it might be better to call this `FeedbackResponse` or smth like tha?, there's some pretty direct overlap here 😅